package/verification-template/gradle/verification-metadata.xml

```diff
@@ -245,6 +245,14 @@
             <sha256 value="d2b115634f5c085db4b9c9ffc2658e89e231fdbfbe2242121a1cd95d4d948dd7" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.google.code.gson" name="gson" version="2.11.0">
+         <artifact name="gson-2.11.0.jar">
+            <sha256 value="57928d6e5a6edeb2abd3770a8f95ba44dce45f3b23b7a9dc2b309c581552a78b" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="gson-2.11.0.pom">
+            <sha256 value="c0e547bea998888e6e25c5886a90e762272bc88b5275343dd2c05ded6ca2e360" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
consider adding a gson exclusion [here](https://github.com/digiholic/osrs-archipelago/blob/34a7bacdda632ae961998562a3982647de17fd5d/build.gradle#L28-L32) so 2.11.0 doesn't need to be added to the verification metadata